metamodel.zenta

```diff
@@ -1,6 +1,155 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
TestData description should be updated according to that we discussed last week

Reply:
I beleive I have done it